app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -125,6 +125,8 @@ class ResponseRedacter
 
         # @param [Hash] body
         def self.redact(hash)
+          return {error: 'TMx response body was empty'} if hash.nil?
+          return {error: 'TMx response body was malformed'} unless hash.is_a? Hash
```

Comment:
let's get some regression specs for this? specifcally in `resolution_proofing_job_spec.rb` if possible?

Reply:
added in [f54f916](https://github.com/18F/identity-idp/pull/6903/commits/f54f916aef5c6266676a0a72545b467d550f836a)